axon-framework/monitoring/tracing.md

```diff
@@ -0,0 +1,535 @@
+# Distributed Tracing
+
+Distributed Tracing enables you to track the path of a message through your system to see how the system behaves and
+performs.
+Axon Framework provides additional tracing functionality to track what takes time in your microservice, such as how long
+it took to load the aggregate, how long the actual command invocation took, or how long it took to publish events.
```

Comment:
What do you think about listing the three options right away here, providing a link to those sections in this file as well.

Reply:
I think that's quite ugly. What I've done, is move the terminology section down. The table with all options should be immediately visible to the user now